### PR TITLE
content_macros/event_fields update css and add a slot for extra fields

### PR DESCRIPTION
**Pull request by cillianderoiste**
Resync the event view with the prototype. Move the description into the body and add an extra slot into the event_fields macro to allow additional fields to be included when customized (which we need for iKath).

### CODE REVIEW

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/220/
Test PASSed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/223/
Test PASSed.

File: src/ploneintranet/workspace/browser/templates/content_macros.pt [hunk 1]
```diff
@@ -53,8 +53,17 @@
...
 
 
-    <metal:event_fields define-macro="event_fields">
-      <fieldset class="vertical"  tal:define="read_only python:False">
+    <metal:event_fields define-macro="event_fields"
+                        tal:define="read_only python:False">
+      <fieldset class="vertical fancy">
+        <label class="description">
+          Description
+          <textarea name="description" placeholder="Enter a description of the task"
```
Comment (mattss) [inline src/ploneintranet/workspace/browser/templates/content_macros.pt]: Should this read 'description of the event'?

Comment (cillianderoiste) [inline src/ploneintranet/workspace/browser/templates/content_macros.pt]: Fixed, thanks!

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/224/
Test PASSed.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/225/
Test PASSed.

**Pull request merged**